loculus_values/values.yaml

```diff
@@ -163,6 +163,7 @@ defaultOrganismConfig: &defaultOrganismConfig
         displayName: NCBI release date
         type: date
         header: "INSDC"
+        includeInDownloadsByDefault: true
```

Comment:
I don't support including the NCBI release date by default. It's not set for any uploads to PPX even if they have been submitted to NCBI, and it is already covered by the `earliestReleaseDate` for stuff submitted to INSDC.

Reply:
This is an odd thing to have changed now - is there a story behind this?

Reply:
@corneliusroemer I removed this for now - we can potentially discuss in a later issue if you would still like to include it :-)